guide/sections/part2/global-services.adoc

```diff
@@ -130,6 +131,7 @@ In WIS2, Global Caches provide access to WMO core data for data consumers. This
 * For messages received on the topic ``++origin/a/+/data/core/#++`` or ``++cache/a/+/data/core/#++``, a Global Cache shall:
 ** If the message contains the property ``"properties.cache": false``,
 *** Republish the message at topic ``cache/a/wis2/...``, matching ``+/a/wis2/...`` where the original message has been received, after having updated the id of the message.
+*** Add the property ``properties.global-cache`` with the value of the centre identifier of the Global Cache to identify the origin of the data source downloaded by downstream applications.
```

Comment:
`global-cache` is only added when `properties.cache: false `?

Reply:
Fixed.